scene/resources/material.cpp

```diff
@@ -878,6 +878,10 @@ void BaseMaterial3D::_update_shader() {
 		code += "\tvec2 base_uv2 = UV2;\n";
 	}
 
+	if (features[FEATURE_HEIGHT_MAPPING] && flags[FLAG_UV1_USE_TRIPLANAR]) {
+		WARN_PRINT("Height mapping is not supported on triplanar materials. Ignoring height mapping in favor of triplanar mapping.");
```

Comment:
Should this be `WARN_PRINT_ONCE`? I guess it can get pretty spammy.

Reply:
The message is printed every time a StandardMaterial3D is compiled, so if you just change an uniform that doesn't cause a shader compilation, it won't reappear. Also, I'll look into amending the message so it mentions the resource name as well.